src/Symfony/Component/CssSelector/XPath/XPathExpr.php

```diff
@@ -46,9 +46,9 @@ public function getElement(): string
     /**
      * @return $this
      */
-    public function addCondition(string $condition): static
+    public function addCondition(string $condition, string $conjunction = 'and'): static
```

Comment:
the argument should be named `operator`, not `conjuction`. When using `or`, you create a disjunction.

Reply:
You're right. Done.